fio/fio_run

```diff
@@ -254,20 +256,154 @@ report_info()
 	echo $1 >> /tmp/log
 }
 
+convert_unit()
+{
+	unit=$1
+	base=1
+
+	if [[ $unit == "KiB/s" ]]; then
+		base=1024
+	elif [[ $unit == "MiB/s" ]]; then
+		base=`echo 1024*1024 |bc`
+	elif [[ $unit == "GiB/s" ]]; then
+		base=`echo 1024*1024*1024 |bc`
+	elif [[ $unit == "sec" ]]; then
+		base=`echo 1000*1000*1000 | bc`
+	elif [[ $unit == "msec" ]]; then
+		base=`echo 1000*1000 | bc`
+	elif [[ $unit == "usec" ]]; then
+		base=1000
+	elif [[ $unit == "nsec" ]]; then
+		base=1
+	else
+		echo unknown unit $unit
+		exit 1
+	fi
+	echo $base
+}
```

Comment:
As stated in pyperf wrapper, why duplicate the unit conversion across multiple wrappers?